ui/components/app-wrapper.js

```diff
@@ -45,5 +47,8 @@ export default function wrapPage(Page, dataFn, headerFooterParams) {
       return { err };
     }
   };
-  return WrappedPage;
+  return withRedux({
+    createStore,
+    debug: process.env.NODE_ENV === 'development',
```

Comment:
Something I've noticed with `NODE_ENV` is that folks seem to usually compare it to `production` rather than checking to see if it's `development`, meaning that this conditional might instead be expressed as `process.env.NODE_ENV !== 'production'`. However, if our docker setup is correctly setting `NODE_ENV` to `development` then I guess it doesn't matter!

Reply:
It's funny you mention that -- I originally had `!== 'production'`, but realized we actually have three environments: production, development, test. Jest sets the latter by default (though I think we make it explicit, too) to trigger a slightly different babel config. When I set to `!== 'production'`, I saw some rather noisy tests, so I opted out. What do you think?

Reply:
Oh, interesting... I am used to testing frameworks that suppress stdout unless the test fails, which I like because that makes failing tests easier to debug without making successful tests super noisy.  If there's no way to enable that in jest, though, I think it's fine to leave your code as-is!